packages/editor/src/components/post-author/index.js

```diff
@@ -21,17 +21,23 @@ function PostAuthor() {
 
 	const { authorId, isLoading, authors, postAuthor } = useSelect(
 		( select ) => {
-			const { getUser, getUsers, isResolving } = select( 'core' );
+			const { __unstableGetAuthor, getAuthors, isResolving } = select(
+				'core'
+			);
 			const { getEditedPostAttribute } = select( 'core/editor' );
-			const author = getUser( getEditedPostAttribute( 'author' ) );
+			const author =
+				postAuthor ||
+				__unstableGetAuthor( getEditedPostAttribute( 'author' ) )[ 0 ];
```

Comment:
Actually, why do we need `[ 0 ]` here. getAuthor only returns one author right?

Reply:
Yes, but it is returned as an array with a single element. I can change this so `__unstableGetAuthor` returns a single element which makes more sense based on the naming, does that sound good?